zebra-state/src/on_disk.rs

```diff
@@ -75,8 +65,22 @@ impl SledState {
         }
     }
 
+    pub(super) fn get_at(&self, height: BlockHeight) -> Result<Option<BlockHeaderHash>, Error> {
```

Comment:
I think we should call this get_main_chain_at
https://github.com/ZcashFoundation/zebra/pull/741/files#r459759219